lighthouse-core/runner.js

```diff
@@ -144,7 +144,8 @@ class Runner {
           score = report.score;
         }
 
-        return {
+        return opts.driver.getChromeVersion().then(version => ({
```

Comment:
I don't think it hurts to save the whole UA string. It might be valuable to later add chromeVersion and  platform (i.e.. operating system) as additional metadata properties.